### DISCUSSION TITLE
Error in localized GraphQL Query

### DISCUSSION THREAD

**Discussion by Dinesh2703** [pimcore/pimcore#8133] [Q&A / Support] [answered]
I am trying to get a particular localized field in another language. I have follwed PIMCore GrapQl query documentation for this as shown in below image.
![sampleQuery](https://user-images.githubusercontent.com/41935464/107482118-45d07f80-6ba5-11eb-9ce7-f4b413bca4c2.PNG)

My query is-
![localizedGraphQL](https://user-images.githubusercontent.com/41935464/107482162-4ec15100-6ba5-11eb-981c-58561fd6b853.PNG)

Its showing error - 
![error](https://user-images.githubusercontent.com/41935464/107482274-787a7800-6ba5-11eb-8d44-ff0d1e61df14.PNG)

Please Suggest the fix. My PIMCore version is v6.8.6

Thanks

**Comment by weisswurstkanone** [comment]
As the documentation indicates, language selection is only supported on request level, not on field level.

**Comment by dvesh3** [comment]
@Dinesh2703 please make sure your field "DisplayName" is localized. you can check Documentation on right side: click query -> StateConnection ->StateEdge -> object_State. e.g.
![image](https://user-images.githubusercontent.com/5137917/107494691-6097f980-6b8f-11eb-8e4d-61ea09e15824.png)

**Reply by Dinesh2703** [reply]
Hi @dvesh3 My field "DisplayName" is localized. But still can not see the "language"  in Object_State.

![image](https://user-images.githubusercontent.com/41935464/107601186-fabd7780-6c4b-11eb-911f-2fdbc5525021.png)

**Comment by Dinesh2703** [comment, marked as answer]
@dvesh3 Thanks, Now I know the reason. The language does not appear on bricks field. It appears only in class field
